src/makeCalculator.js

```diff
@@ -37,7 +37,35 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(a) {
+      calculator.result += a;
+
+      return calculator.result;
+    },
+    subtract(a) {
+      calculator.result -= a;
+    },
```

Comment:
Add default values and blank lines
why in one case you're returning smth and in other dont?
let's be consistent
```suggestion
  const calculator = {
    result: 0,
    
    add(a = 0) {
      calculator.result += a;

      return calculator.result;
    },
    
    subtract(a = 0) {
      calculator.result -= a;
    },
```